src/components/core/InteractiveLink/InteractiveLink.tsx

```diff
@@ -81,17 +95,59 @@ export default function InteractiveLink({
   return null;
 }
 
-export const StyledAnchor = styled.a<{ disabled?: boolean }>`
-  color: ${({ disabled }) => (disabled ? colors.porcelain : colors.shadow)};
+export function InteractiveLinkNeedsVerification({
+  inheritLinkStyling,
+  href,
+  children,
+}: {
+  inheritLinkStyling: boolean;
```

Comment:
this prop is listed as required here, but optional above in the regular `InteractiveLink`. let's keep it optional and default to false?